src/styles/_nav.scss

```diff
@@ -0,0 +1,60 @@
+.navbar {
+  border: 2px solid rgb(255, 255, 255);
+  box-shadow: 5px 5px 10px 0 #000;
+  display: flex;
+  gap: 120px;
+  flex-direction: row;
+  background-color: #fff;
+  padding: 15px;
+
+  h1 {
+    color: rgb(137, 54, 216);
+    background-color: #fff;
+    margin-left: 20px;
+    margin-top: 5px;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  }
+
+  .nav-links {
+    font-size: 30px;
+    display: flex;
+    gap: 30px;
+    flex-grow: 1;
+    background-color: #fff;
+
+    .link-link {
+      background-color: #fff;
+
+      :hover {
+        font-size: 18px;
+        color: random($limit: 9);
+      }
+    }
+
+    .link-linked {
+      background-color: #fff;
+      font-size: 16px;
+      padding-bottom: 2px;
+      border-bottom: 1px solid aqua;
```

Comment:
- [ ] :red_circle: I think you can remove the background property as it makes you repeat yourself. The parent element has its background set to white `#ffffff`. This child will inherit this prop from the parent container.
- [ ]  :red_circle: Also there is no need for the border-bottom property as the template doesn't apply any border on these elements. Kindly remove this property to match the template. Remember to remove the underline text-decoration which add a bottom line to these elements
- [ ] :red_circle: Kindly notice :arrow_down:  your font color applied on these elements  differs from the one in the Template


Your Design
![Screenshot from 2023-01-27 14-48-47](https://user-images.githubusercontent.com/26736582/215093175-4ab7f3b2-d7c9-42f7-8a22-b54aaf368d15.png)

Template Design 

![Screenshot from 2023-01-27 14-49-22](https://user-images.githubusercontent.com/26736582/215093207-5c35c505-16ff-4dad-be29-3e2950b8011c.png)

Reply:
Applied the suggested changes